content/kubermatic/v2.16/concepts/kubermaticconfiguration/_index.en.md

```diff
@@ -14,5 +14,5 @@ The following is an example configuration, showing all possible options. Note th
 don't define explicitly are always defaulted to these values.
 
 ```yaml
-{{< readfile "kubermatic/master/data/kubermaticConfiguration.yaml" >}}
+{{< readfile "kubermaticConfiguration.yaml" >}}
```

Comment:
I mean, now you need to move `data/kubermatic/master/data/kubermaticConfiguration.yaml` to `content/kubermatic/v2.16/concepts/kubermaticconfiguration/`

And do the same operation with all other files that we can see in this patch.

Reply:
Are you sure? I mean the master is likely to be changed especially for v2.16 right?
I think i just update those files with the one in the KKP repository and their corresponding release branch, which should be the source of truth.